MuteSpotifyAds/SpotifyManager.swift

```diff
@@ -8,7 +8,7 @@
 
 import Cocoa
 
-func shell(launchPath: String, arguments: [String]) -> String
+func runCommand(launchPath: String, arguments: [String]) -> String
```

Comment:
If I see it correctly, this function is not used anywhere, therefore I'm going to remove it.

Reply:
Furthermore we a have a similar function `runAppleScript(...)`, so if we would need a `runCommand(...)` function, it would be good to re-use code there.